maestro-cli/src/main/java/maestro/cli/api/ApiClient.kt

```diff
@@ -368,6 +386,25 @@ class ApiClient(
         }
     }
 
+    private fun startTrial(authToken: String, baseUrl: String): Boolean {
+        println("Starting your trial...")
+        val body = "".toRequestBody(null)
```

Comment:
Weird why is the request for this null? It can be the default to empty body IMO.